src/Guild.js

```diff
@@ -0,0 +1,48 @@
+const RateLimiter = require('./RateLimiter');
+
+class Guild {
```

Comment:
Why are all functions static? You could create guild objects with the guild id etc., optionally with a get function and some form of cache.